test/TestServerProjectsLowLevel/dpg-customization/Models/ProductResult.Serialization.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace dpg_customization_LowLevel.Models
+{
+    public partial class ProductResult
```

Comment:
This type should not be part of the public API.

Reply:
Do we have any criterion? I see there has [case ](https://github.com/azure/autorest.csharp/blob/feature%2Fv3/test/TestServerProjects/paging/Generated/Models/ProductProperties.Serialization.cs#L13)which is serialization but still public.

Reply:
Per the decision in the DPG design meeting yesterday, we'll make this type internal.